core/block_validator.go

```diff
@@ -64,6 +64,15 @@ func (v *BlockValidator) ValidateBody(block *types.Block) error {
 		}
 		return consensus.ErrPrunedAncestor
 	}
+	if v.config.IsGingerbreadP2(block.Number()) {
+		bytesBlock := new(BytesBlock).AddBytes(params.MaxTxDataPerBlock)
+
+		for _, tx := range block.Transactions() {
+			if err := bytesBlock.SubBytes(uint64(tx.Size())); err != nil {
+				return err
```

Comment:
Above you log an error here, maybe add this here as well?

Reply:
The block validator response is logged later.
In the proposal, we are not using this same instance, that's why we log it in every case